finance/proposal-calls/cycle3/Aperio-Infrastructure-Maintenance.md

```diff
@@ -0,0 +1,98 @@
+## Infrastructure and core/coordinated package maintenance
+
+### Project team
+
+This proposal is to fund the following people at
+[Aperio Software](https://aperiosoftware.com):
+
+* Thomas Robitaille
+* Stuart Mumford
+* Drew Leonard
+
+This could also fund other individuals working at Aperio Software if needed.
+
+### Project Summary
+
+This project covers three main areas of work:
+
+* Maintenance of project infrastructure (such as pytest and sphinx plugins,
+  extension-helpers, continuous integration, wheel building, and so on). As
+  illustrated in
+  [astropy/astropy-project#118](https://github.com/astropy/astropy-project/issues/118),
+  the infrastructure for the project has grown significantly over the last few
+  years, and requires regular maintenance.
+* Maintenance and development of the
+  [astropy.timeseries](https://docs.astropy.org/en/stable/timeseries/) and
+  [astropy.visualization.wcsaxes](https://docs.astropy.org/en/stable/visualization/wcsaxes/)
+  sub-packages as well as the [reproject](http://reproject.readthedocs.io) and
+  [astropy-healpix](https://astropy-healpix.readthedocs.io/en/latest/)
+  coordinated packages.
+
+### Project / work
+
+#### Infrastructure work
+
+We will participate in the maintenance of the existing project infrastructure,
```

Comment:
While we can really use the help, the infrastructure side continues to struggle (i.e., issues over at pytest and doctest plugins continue to linger on). I wonder if Aperio is getting stretched too thin across too many things. I see you have proposed multiple stuff as separate issues.

Reply:
Speaking just for me, this funding request is really for us to continue doing what I can, as I have been. Other things I am working on (mainly not Astropy work) have to take priority most of the time.